src/index.ts

```diff
@@ -139,10 +139,16 @@ class MultinetAPI {
   }
 
   public renameWorkspace(workspace: string, name: string): AxiosPromise {
-    return this.client.axios.put(`workspaces/${workspace}/name`, null, { params: { name }});
+    return this.client.axios.put(`workspaces/${workspace}/name`, null, { params: { name } });
   }
 
-  public async uploadTable(workspace: string, table: string, options: FileUploadOptionsSpec): Promise<Array<{}>> {
+  public async uploadTable(
+    workspace: string,
+    table: string,
+    options: FileUploadOptionsSpec,
+    key?: string,
+    overwrite?: boolean,
```

Comment:
These options should be part of the `options` argument (and added to `FileUploadOptionsSpec` as optional parameters there).